code/__DEFINES/traits/declarations.dm

```diff
@@ -665,6 +665,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// Trait that determines whether our mob gains more strength from drinking during a fist fight
 /* #define TRAIT_DRUNKEN_BRAWLER "drunken brawler" */
 
+/// Makes user able to absorb food through any exposed body part.
+#define TRAIT_FOOD_ABSORBTION "food_absorbtion"
```

Comment:
[isn't the correct spelling "absorption"?](https://brians.wsu.edu/2023/03/28/absorbtion)